author/backends.py

```diff
@@ -39,14 +43,21 @@ class AuthorDefaultBackend(object):
 
     def __init__(self):
         required_middleware = 'author.middlewares.AuthorDefaultBackendMiddleware'
-        if required_middleware not in settings.MIDDLEWARE_CLASSES and required_middleware not in settings.MIDDLEWARE:
-            raise ImproperlyConfigured(
+        new_required_middleware = 'author.middlewares.AuthorDefaultBackendMiddlewareNewStyle'
+        if django.VERSION > (1, 10):
+            if new_required_middleware not in settings.MIDDLEWARE_CLASSES and new_required_middleware not in settings.MIDDLEWARE:
+                raise ImproperlyConfigured(
+                    'Error "%s" is not found in MIDDLEWARE_CLASSES nor MIDDLEWARE. '
+                    'It is required to use AuthorDefaultBackend' % new_required_middleware)
+        else:
+            if required_middleware not in settings.MIDDLEWARE_CLASSES and required_middleware not in settings.MIDDLEWARE:
+                raise ImproperlyConfigured(
```

Comment:
This seems to add unnecessary duplicity. The condition and the `raise` clause seems to be copied. Wouldn't be easier to write something like this?
```
if django.VERSION >= (1, 10):
    required_middleware = 'author.middlewares.AuthorDefaultBackendMiddleware'
else:
    required_middleware = 'author.middlewares.AuthorDefaultBackendMiddlewareNewStyle'
```